src/iiif/queue_zip_consumer.py

```diff
@@ -107,7 +119,7 @@ def process_message(self, message):
                     image_info["url_info"],
                     fail_reason,
                     metadata,
-                    record["request_meta"],
+                    None,  # TODO: Remove parameter because not needed anymore (was: record["request_meta"])
```

Comment:
Deze zou ik meteen verwijderen. Is niet veel werk en haalt meteen troep weg.

Reply:
Staat al een ticket om dit allemaal op te ruimen. Lijkt me  fijner om het als deel daarvan te doen